variables.tf

```
@@ -1,25 +1,36 @@
 variable "awp_cloud_account_id" {
-    description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
-    type        = string
+  description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
+  type        = string
+}
+
+variable "awp_hub_external_account_id" {
```

rename it to awp_centralized_cloud_account_id
```suggestion
variable "awp_centralized_cloud_account_id" {
```

---

renamed